packages/create/templates/hello-nextjs/README.md

```diff
@@ -40,10 +40,20 @@ npm run start
 
 To see that it's working, visit this URL in your browser: [`http://localhost:3000/`](http://localhost:3000/).
 
-Click on the "Run DBOS Workflow" button.
+Click on the "Start Background Job" button.
+
+You should this message updated: `Your background task has completed step 0 of 9.`
+As the background job completes steps, the step counter should go up by one!
+
+Click on the "Crash the application".
+The step counter stops increasing.
+On the command line, you will see that the application has stopped.
+Start the application again.
+```bash
+npm run start
+```
+In the browser, you will see that the execution of steps, resumes where it left off and completes.
```

Comment:
In the browser, you will see that the execution of steps resumes where it left off and eventually completes.